src/reportWebVitals.js

```diff
@@ -0,0 +1,13 @@
+// const reportWebVitals = onPerfEntry => {
+//   if (onPerfEntry && onPerfEntry instanceof Function) {
+//     import('web-vitals').then(({ getCLS, getFID, getFCP, getLCP, getTTFB }) => {
+//       getCLS(onPerfEntry);
+//       getFID(onPerfEntry);
+//       getFCP(onPerfEntry);
+//       getLCP(onPerfEntry);
+//       getTTFB(onPerfEntry);
+//     });
+//   }
+// };
+
+// export default reportWebVitals;
```

Comment:
remove unused code

Reply:
I plan to uninstall before deploy project.